src/background/controller/wallet.ts

```diff
@@ -975,6 +988,12 @@ export class WalletController extends BaseController {
         return Promise.resolve({
           price: { last: '1.0', change: { percentage: '0.0' } },
         });
+      case 'dust':
```

Comment:
Why we need this ? It looks like no extendable. 
Please use token contract name instead of token name.